microsetta_public_api/api/microsetta_public_api.yml

```diff
@@ -384,6 +433,60 @@ components:
             - "sample1"
             - "sample2"
             - "sample3"
+    sampleDataTable:
+      type: object
+      required:
+        - data
+        - columns
+      properties:
+        data:
+          type: array
+          items:
+            type: object
+            required:
+              - sampleId
+            properties:
+              sampleId:
+                $ref: '#/components/schemas/sampleId'
+            example:
+              - {
+                  "sampleId": "sample1",
+                  "Kingdom": "Bacteria",
+                  "Genus": "Bacteroides"
+                }
+              - {
+                  "sampleId": "sample1",
+                  "Kingdom": "Bacteria",
+                  "Genus": "Clostridium"
+                }
+              - {
+                  "sampleId": "sample2",
+                  "Kingdom": "Bacteria",
+                  "Genus": "Bacteroides"
+                }
+              - {
+                  "sampleId": "sample2",
+                  "Kingdom": "Bacteria",
+                  "Genus": null
+                }
+              - {
+                  "sampleId": "sample3",
+                  "Kingdom": "Bacteria",
+                  "Genus": "Clostridium"
```

Comment:
I think this should include `relative-abundance` as well

Reply:
The spec right now technically does not preclude a relative abundance field, though it does not require it either.

Reply:
Which should change if what you want is an endpoint that specifically returns relative abundance